front/app/components/Form/Components/Controls/LinearScaleControl.tsx

```diff
@@ -36,20 +35,98 @@ const LinearScaleControl = ({
 }: ControlProps) => {
   const isSmallerThanPhone = useBreakpoint('phone');
   const theme = useTheme();
-  const maximum = schema?.maximum;
+  const { formatMessage } = useIntl();
+
+  const minimum = 1;
+  const maximum = schema?.maximum ?? 7; // Seven since the maximum number of options is 7
+  const totalOptions = maximum;
   const answerNotPublic = uischema.options?.answer_visible_to === 'admins';
+  const sliderRef = useRef<HTMLDivElement>(null);
+
+  const getValueText = useCallback(
+    (value: number, total: number) => {
```

Comment:
Could we call this getAriaValueText, just to make it clear this isn't a visible label/text? :)